content/en/docs/best-practices/channel-naming.md

```diff
@@ -96,25 +96,37 @@ spec:
   sourceNamespace: my-operators
 ```
 
-The Subscription is providing hints to OLM which are used to determine exactly which version of an operator will get deployed onto the cluster, in this example OLM will look for an operator to deploy that belongs to the candidate channel within a specified catalog source.  
+The [Subscription][subscription] is providing hints to OLM which are used to determine exactly which version of an operator will get deployed onto the cluster, in this example OLM will look for an operator to deploy that belongs to the candidate channel within a specified catalog source.  
 
-Note that exactly which operator version is deployed can depend on more than what you specify in the Subscription.  On initial install, OLM will always attempt to install whatever is the head of the specified channel by default.  Settings within the operator’s CSV also are used by OLM to determine exactly which operator version OLM will deploy or upgrade.
+Note that exactly which operator version is deployed can depend on more than what you specify in the [Subscription][subscription].  On initial install, OLM will always attempt to install whatever is the head of the specified channel by default.  Settings within the operator’s CSV also are used by OLM to determine exactly which operator version OLM will deploy or upgrade.
 
 ## NAMING
 
-Channel names are used to imply what form of upgrade you want to offer for your operator.  For example, you might have an operator that has a candidate or alpha version which is not supported as well as a version where support is offered.
+Channel names are used to imply what form of upgrade you want to offer for your operator. For example, you might have an operator with a candidate release version that is not supported and a version where support is offered.
+		
+
+Our motivation with this convention is to encourage a better user experience for OLM users to have an intuitive 
+understanding of the level of maturity and supportability of the solutions that will get installed.
+It can get very confusing for cluster admins and Operator consumers in general when each solution distributed in the OLM catalog adopts different names or different meanings for the same terminologies.
```

Comment:
"It can be very confusing for cluster admins and Operator consumers in general when each solution distributed in the OLM catalog adopts different names or different meaning for the same terminologies" 

Is this sentence really required in this doc? We know it's out motivation to write this entire channel naming convention doc, but just reading the sentence it looks like it has the potential to throw off users with words like "solution", "terminologies" etc. We know what they mean to us who's been looking at these "solutions" and "terminoliges" for a long time now but a new operator author will probably be very confused with this sentence (and thereby increase the cognitive overload needed for reading this doc)

Reply:
One of the goals is to clarify why we need the convention.
Besides, it shows obvious for us that have not been obvious to everybody.